client/src/app/KeyboardBindings.js

```diff
@@ -175,7 +198,29 @@ export default class KeyboardBindings {
     this.undo = findUndo(menu);
     this.redo = findRedo(menu);
 
+    menu = this.updateRemoveSelectionEntry(menu);
+
     this.updateCustomEntries(menu);
+
+    return menu;
+  }
+
+  updateRemoveSelectionEntry(menu) {
+
+    // (1) set primary shortcut depending on platform
+    const primaryAccelerator = this.isMac ? 'Backspace' : 'Delete';
+
+    menu = findAndReplaceAll(
+      menu,
+      ({ accelerator }) => isAccelerator(accelerator, 'Delete'),
+      { accelerator: primaryAccelerator }
+    );
+
+    // (2) register it
+    this.removeSelection =

```

Comment:
Maybe it makes sense to move the whole method contents up, i.e. place them into `#update`? I'd assume that at least this line is handled where all other actions are registered (in `#update`).

Reply:
We could, but we handle [custom entries similar](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/KeyboardBindings.js#L226), outside of `#update`.

Reply:
Fine then :)